src/librustc/ty/maps/plumbing.rs

```diff
@@ -761,6 +761,7 @@ pub fn force_from_dep_node<'a, 'gcx, 'lcx>(tcx: TyCtxt<'a, 'gcx, 'lcx>,
         DepKind::EraseRegionsTy |
         DepKind::NormalizeTy |
         DepKind::SubstituteNormalizeAndTestPredicates |
+        DepKind::InstanceDefSizeEstimate |
```

Comment:
Is this the correct location? Or should it `force!` something like some of the queries below?

Reply:
Yes that's the correct location. Very good! `:)`

`force_from_dep_node` takes a `DepNode` and tries to execute the query corresponding to it. However, a `DepNode` only consists of a `DepKind` and an opaque hash value (which acts as a UUID). It is only in special cases, like when the hash value is actually a `DefPathHash` that we can look up in a table, that we can map from `DepNode` to the correct query and its arguments. For `ty::Instance` we don't have such a table that would make it possible to do the mapping.